stdlib/tkinter/ttk.pyi

```diff
@@ -1158,18 +1158,27 @@ class Treeview(Widget, tkinter.XView, tkinter.YView):
     def tag_has(self, tagname: str, item: str | int) -> bool: ...
 
 class LabeledScale(Frame):
-    label: Incomplete
-    scale: Incomplete
-    # TODO: don't any-type **kw. That goes to Frame.__init__.
```

Comment:
Can you add comments, both here and to `Frame.__init__`, that the arguments are same as for `Frame.__init__` and must be kept in sync? I'm worried that this will get outdated when `Frame.__init__` is changed.